LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>In the future, please import these functions from the existing package instead of copy pasting!!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>done:)</p>